codecov.yml

```diff
@@ -4,5 +4,6 @@ ignore:
   - "**/*_Mock.swift"
   - "**/*_Vendor.swift"
   - "**/Generated/*.swift"
+  - "stream-chat-swift-docs/**/*"
```

Comment:
Why is this needed? 🤔

Reply:
https://getstream.slack.com/archives/CRML30N0H/p1617104227241200?thread_ts=1617102442.236200&cid=CRML30N0H

Reply:
Docs are not tested :D My doc PR reported decrease in coverage